Added IPFS Shared metadata doc (#93)
cennznet/wiki: Dapp-development/Guides/NFT-metadata-standard.md

e.g. `ipfs://bafybeigdyrzt5sfp7udm7hu76uh7y26nf3efuylqabf3oclgtqy55fbzdi/123.json` returns the metadata JSON document for token with serial number 123

### 3) IPFS shared

> All NFTs in the series share the same metadata and this file should be uploaded as an IPFS directory.

Reviewer: rephrase
```suggestion
All NFTs in the series share the same metadata and the file is hosted on ipfs
```

> When fetching the metadata CID from any token in the series, it will return the same uri, pointing to the ipfs directory for the whole series

Reviewer: ```suggestion
When fetching the metadata CID from any token in the series, CENNZnet will return the same uri, pointing to the shared ipfs metadata file
```
